packages/tools/pie-monorepo-utils/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# @justeattakeaway/git-hooks-scripts
+# @justeattakeaway/pie-monorepo-utils
 
 ## 0.2.0
```

Comment:
I guess this version needs reverting if we're going back to 0.0.0?

Reply:
Weird, I did update this file but must'ver everted with the rebase.